lib/protobuf/field/message_field.rb

```diff
@@ -9,7 +9,7 @@ class MessageField < BaseField
       #
 
       def acceptable?(val)
-        unless val.instance_of?(type_class) || val.respond_to?(:to_hash)
+        unless val.is_a?(type_class) || val.is_a?(Hash)
```

Comment:
hmmm, if I move this to `unless val.is_a?(type_class) || val.respond_to?(:to_hash)`, then  this [test](https://github.com/localshred/protobuf/pull/252/files#diff-1de145c39faa7b92fc0a33ea3d9fc49bR69) fails.

I put that test in there because assigning an unrelated object to a MessageField and having it pass (and possibly drop fields silently if they don't match) seems wrong.  But if that is ok or the desired behavior I'll go ahead and make the change and remove the test.